hathor/manager.py

```diff
@@ -633,6 +640,10 @@ def _resume_validations(self) -> None:
         """ This method will resume running validations that did not run because the node exited.
         """
         assert self.tx_storage.indexes is not None
+        if not self.tx_storage.indexes.deps:
+            self.log.warn('deps index disabled')
```

Comment:
Why warn? I feel we should either use debug or exit the process.

Reply:
I think debug makes more sense.

Reply:
Removed this, left the assert as per the next comment.

Reply:
Also moved to #275